src/ucs/algorithm/levenshtein_dist.c

```diff
@@ -13,36 +13,37 @@
 #include <ucs/sys/sys.h>
 #include <string.h>
 
-#define MAX_ENV_VAR_LENGTH 30
+#define MAX_UCX_ENV_VAR_LENGTH 30
 
-int ucs_levenshtein(const char *s1, const char *s2)
+int ucs_levenshtein(const char *str1, const char *str2)
 {
-    uint32_t s1len = strlen(s1);
-    uint32_t s2len = strlen(s2);
-    uint32_t x, y;
-    static uint32_t matrix[MAX_ENV_VAR_LENGTH][MAX_ENV_VAR_LENGTH];
+    uint32_t len1 = strlen(str1);
+    uint32_t len2 = strlen(str2);
+    static uint32_t dists[MAX_UCX_ENV_VAR_LENGTH][MAX_UCX_ENV_VAR_LENGTH];
+    uint32_t i, j;
 
-    if((s1len > MAX_ENV_VAR_LENGTH) || (s2len > MAX_ENV_VAR_LENGTH)) {
+    if ((len1 > MAX_UCX_ENV_VAR_LENGTH) || (len2 > MAX_UCX_ENV_VAR_LENGTH)) {
         return -1;
     }
 
-    matrix[0][0] = 0;
-    for (x = 1; x <= s2len; ++x) {
-        matrix[x][0] = matrix[x-1][0] + 1;
+    dists[0][0] = 0;
+    for (i = 1; i <= len2; ++i) {

```

Comment:
if `len2 == MAX_UCX_ENV_VAR_LENGTH`, we'll reach out-of-bounds `dists[MAX_UCX_ENV_VAR_LENGTH]`